The userlib_execdir was not getting set correctly, the quoting
was wrong as these quoted items were not getting expanded correctly.

Upstream-Status: Pending

Signed-off-by: Saul Wold <sgw@linux.intel.com>

Index: util-linux-2.24.1/configure.ac
===================================================================
--- util-linux-2.24.1.orig/configure.ac
+++ util-linux-2.24.1/configure.ac
@@ -76,7 +76,7 @@ fi
 AC_SUBST([usrsbin_execdir])
 
 AS_CASE([$libdir],
- ['${exec_prefix}/'* | '${prefix}/'* | /usr/*],
+ [${exec_prefix}/* | ${prefix}/* | /usr/*],
    [usrlib_execdir=$libdir],
  [usrlib_execdir='${exec_prefix}'$libdir]
 )
